adiff/adiff.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Company>Microsoft Corporation</Company>
     <Copyright>2020 Microsoft Corporation</Copyright>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Instead of doing this, can we use `$(RollForward)` `Major`?

See: https://github.com/jonathanpeppers/boots/commit/7f4a336259c4fc37202ea5d1f8dea2b0e1c5a1c7

It would be nice if this tool could keep running on .NET Core 3.1, 5, 6, etc. When .NET Core 3.1 is out of support, we could drop it then.

Reply:
Ooh I didn't know about this, I like this approach.